configuration/configurator/schemas/SchemaAttributes.cpp

```diff
@@ -105,6 +105,10 @@ void CAttribute::getDocumentation(StringBuffer &strDoc) const
             return; // HIDDEN
         else
             pToolTip = pAppInfo->getToolTip();
+
+        const char* pReq = pAppInfo->getRequired();
+        if (pReq != nullptr && stricmp("True", pReq) == 0)
+            pRequired = "required";
```

Comment:
rather than using the literal "required" here, why don't you use TAG_REQUIRED, or XML_ENV_VALUE_REQUIRED